### DISCUSSION TITLE
Error: Failed to build Liquidsoap

### DISCUSSION THREAD

**Discussion by jorgefduran** [savonet/liquidsoap#2921] [Installation]
Hi, I have a problem with a fresh centovacast installation, after run the install script I get:

| Post-installation Errors
|
| Note - the following problems occurred during post-installation:
|
| - liquidsoap did not install successfully and will NOT be available for use.
|   To try again, run:
|   /usr/local/centovacast/sbin/update --add liquidsoap

When run:

/usr/local/centovacast/sbin/update --add liquidsoap

The error is:

Building Liquidsoap ...
[NOTE] Pinning unchanged
samplerate is now pinned to version 0.1.5

Your request can't be satisfied:
- unmet availability conditions, e.g. false

[NOTE] Pinning command successful, but your installed packages may be out of sync.
! Error: Failed to pin libsamplerate package version
! Error: Installation failed
Liquidsoap installation failed
! Error: Installer exited with error, aborting

Anyone knows how to solve this?

OS CentOS 7 64 bit

**Comment by toots** [comment]
Hi! Thanks for reporting. This seems like an error with centovacast to me..
